### ISSUE TITLE
Feature request: include a code of conduct

### ISSUE THREAD

**Issue by almndbtr** [neume-network/documents#8]
## Summary

In [Social Architecture's Safety](https://github.com/neume-network/documents/blob/main/social-architecture.md#safety), it reads:

> All neume network code repositories must include a CODE_OF_CONDUCT.md document to allow contributing safely.

Since this repository is under the neume-network, I think it should also include a code of conduct?

I recognize how "meta" of a request this is 😉 but I'll leave this open until one is introduced! ✌️

**Comment by TimDaub** [comment]
> Since this repository is under the neume-network, I think it should also include a code of conduct?

makes sense. It must include a CODE_OF_CONDUCT.md and currently there's none.
